### DISCUSSION TITLE
ERROR : update_addressable failed in device-ble.

### DISCUSSION THREAD

**Discussion by HCL-BMC** [edgexfoundry/community#87] [General]
device-ble binary is generated from "https://github.com/edgexfoundry-holding/device-bluetooth-c"

I am working on adding new feature support to device-ble from the below source
"https://github.com/edgexfoundry-holding/device-bluetooth-c"  from main branch.

For this, I setup device-sdk v1.3.1 and edgex-go v2. 
All edgex micro services are running. 

http://localhost:59881/api/v2/addressable -> This URL is not valid.
So, getting this " update_addressable failed" issue.

is this a valid URL ? Any suggestions to resolve this issue ?
Thanks in Advance.

**Comment by HCL-BMC** [comment]
I switched edgex-go Hanoi branch(v1.3.0) and all the micro services are running. As it supports v1 and v2 egdex services.
Then, I changed to device-sdk-c and run the device-ble binary.  This error got resolved.

Facing another REST server connection errors.
"level=ERROR ts=2023-04-10T04:56:51Z app=device-ble msg="HTTP server: unable to resolve bind address edgex-device-ble"
level=ERROR ts=2023-04-10T04:56:51Z app=device-ble msg="Unable to start HTTP server"

Any suggestions on this error ?

**Comment by bnevis-i** [comment]
> "level=ERROR ts=2023-04-10T04:56:51Z app=device-ble msg="HTTP server: unable to resolve bind address edgex-device-ble"

What is the host name of the container running this process?  Is it `edgex-device-ble`?

**Reply by HCL-BMC** [reply]
I am running all this process in my local machine only. Changed the host to "localhost" resolved this issue. 

Thanks for your response and suggestions.
